src/main/java/gift/controller/ProductController.java

```diff
@@ -29,35 +31,33 @@ public ProductController(ProductService productService) {
     //상품 조회
     @GetMapping("/products/{productId}")
     public ResponseEntity<ProductResponseDto> getProduct(@PathVariable long productId) {
-        return new ResponseEntity<>(productService.getProduct(productId).toResponseDto(),
+        return new ResponseEntity<>(
```

Comment:
이번 미션에서 진행하신 내용은 아니지만 chaned에서 보여서 첨언을 남겨봐요. 😃

ResponseEntity에는 편의를 위한 정적 팩토리 메서드들이 구현되어있는데요. 

지금 같은 경우에는 ResponseEntity.`ok` 메서드를 대신 활용해볼 수 있을 것 같아요. 😃
```suggestion
        return ResponseEntity.ok(productService.productToResponseDto(productService.getProduct(productId)));
```
이 외에도 어떤 메서드들이 있는지 확인해보고 괜찮다고 생각되신다면 이 정적 팩토리 메서드를 활용하도록 응답을 변경해줘도 좋을 것 같아요. 😉 

<img width="791" alt="image" src="https://github.com/user-attachments/assets/dcb7480e-3dc9-45fb-8a55-2ce3aecd72ad" />